docs/overview/queries/elastic_query_range.md

```diff
@@ -3,4 +3,52 @@ id: elastic_query_range
 title: "Range Query"
 ---
 
-TBD
+A query that matches documents that contain terms within a provided range.
+
+In order to use the `Range` query import following:
+```scala
+import zio.elasticsearch.query.RangeQuery
+import zio.elasticsearch.ElasticQuery._
+```
+
+You can create a `Range` query using the `range` method this way:
+```scala
+val query: RangeQuery = range(field = "intField")
+```
+
+You can create a type-safe `Range` query using the `range` method this way:
+```scala
+val query: RangeQuery = range(field = Document.intField)
+```
+
+If you want to change `boost`, you can use the `boost` method:
+```scala
+val queryWithBoost: RangeQuery = range(field = Document.intField).boost(2.0)
+```
+
+If you want to change `format`, you can use the `format` method:
+```scala
+val queryWithFormat: RangeQuery = range(field = Document.dateField).format("uuuu-MM-dd")
```

Comment:
Is this the correct date format?

Reply:
I think yes, I have copied it from our tests.